### ISSUE TITLE
Fetal pleural effusion

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7218]
**Preferred term label:**
Fetal pleural effusion

**Synonyms**
Subcategories
--Fetal hydrothorax
(https://pubmed.ncbi.nlm.nih.gov/32926417/)
--Fetal chylothorax 
(https://pubmed.ncbi.nlm.nih.gov/28351595/)

-- exudative (chylo)and transudative (hydrothorax), no separate terms for now.

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by patt1morgan** [comment]
@pnrobinson 
Definition:  Fetal pleural effusion is the accumulation of excess fluid in the layers of tissue (pleura) lining the lungs and wall of the chest.  It may be primary, also termed hydrothorax,  occurring as an isolated finding or it may be secondary, most commonly resulting from non-immune hydrops.  PMID: 18280793 (also includes images of pleural effusion)

Comment: With the ongoing evolution and more widespread use of obstetric ultrasound, fetal pleural effusions are being recognized more often and at earlier gestational ages. 

??Add Primary pleural effusion and Secondary pleural effusion as 'children' of the term.

**Comment by pnrobinson** [comment]
added
